exotica_core/include/exotica_core/problems/dynamic_time_indexed_shooting_problem.h

```diff
@@ -64,6 +67,14 @@ class DynamicTimeIndexedShootingProblem : public PlanningProblem, public Instant
     Eigen::MatrixXd get_Q(int t) const;               ///< Returns the precision matrix at time step t
     void set_Q(Eigen::MatrixXdRefConst Q_in, int t);  ///< Sets the precision matrix for time step t
 
+    Eigen::MatrixXd get_Qf() const;             ///< Returns the cost weight matrix at time N
+    void set_Qf(Eigen::MatrixXdRefConst Q_in);  ///< Sets the cost weight matrix for time N
+
+    Eigen::MatrixXd get_R() const;  ///< Returns the control weight at time step t
+    DynamicsSolverPtr get_dynamics_solver() const;
+
+    int get_num_controls() const;  ///< Returns size of control vector
```

Comment:
Why? The PlanningProblem should already have that

Reply:
It's not even implemented :D